react/week3/todos/src/components/todolist.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+
+class Form extends React.Component {
+    render() {
+      return (
+        <div>
+          <form onSubmit={this.props.handleSubmit}>
+            <label>Todo description:
+              <input value={this.props.description} onChange={this.props.handleDescriptionChange} name="description" required /><br />
+            </label>
+            <label>Deadline:
+              <input type="date" value={this.props.deadline} onChange={this.props.handleDeadlineChange} name="deadline" required /><br />
+            </label>
+            <input type='submit' value='Add todo' />
+          </form>
+        </div>
+      )
+    }
+}
+
+function FancyBorder(props) {
+  return (
+    <div className={'FancyBorder'}>
```

Comment:
This can be written as: 
```suggestion
    <div className="FancyBorder">
```

It's also generally best practise to write classes using kebab-case in css: `fancy-border`